apps/dotcms-ui/src/app/view/components/dot-edit-layout-designer/dot-edit-layout-designer.component.ts

```diff
@@ -214,11 +201,12 @@ export class DotEditLayoutDesignerComponent implements OnInit, OnDestroy, OnChan
                 sidebar: this.createSidebarForm()
             })
         });
-        this.form.valueChanges.pipe(takeUntil(this.destroy$)).subscribe(() => {
-            this.isModelUpdated = !_.isEqual(this.form.value, this.initialFormValue);
+        this.form.valueChanges.pipe(takeUntil(this.destroy$), debounce(() => interval(300))).subscribe(() => {
```

Comment:
why `interval` and not `timer`?

Reply:
there's also the `debounceTime` operator
https://stackblitz.com/edit/typescript-adheqt?file=index.ts&devtoolsheight=50